pyproject.toml

```diff
@@ -42,28 +41,54 @@ classifiers = [
 
 [tool.poetry.dependencies]
 python = "~3.6 || ~3.7 || ^3.8"  # Compatible python versions must be declared here
+pyang = "^2.4.0"
+pyangbind = {git = "https://github.com/ahmet2mir/pyangbind.git"} # https://github.com/robshakir/pyangbind/pull/233/files
```

Comment:
This is my fork to allow using python 3 where pyangbind uses enum which is not available for python3 https://github.com/ahmet2mir/pyangbind/blob/master/requirements.txt#L6